security/us-east-1/security-compliance --/awsinspector.tf

```diff
@@ -0,0 +1,52 @@
+resource "null_resource" "default_resources" {
+  provisioner "local-exec" {
+    command = "aws inspector2 update-organization-configuration --auto-enable ec2=false,ecr=false --profile ${var.profile} --region ${var.region}"
+  }
+}
+
+###
+# Admin account config
+###
+resource "null_resource" "enable_admin_account" {
+  # Strategy will be to enable the least expensive service, where there are less existing resources/less activity.
+  provisioner "local-exec" {
+    command = "aws inspector2 enable --account-ids ${var.accounts.security.id} --resource-types ECR --profile ${var.profile} --region ${var.region}"
+  }
+}
+
+# resource "null_resource" "disable_admin_account" {
+#   provisioner "local-exec" {
+#     command = "aws inspector2 disable --account-ids ${var.accounts.security.id} --resource-types ECR EC2 --profile ${var.profile} --region ${var.region}"
+#   }
+# }
+
+###
+# Member account config, first associate next enable.
+###
+resource "null_resource" "associate_account" {
+  provisioner "local-exec" {
+    command = "aws inspector2 associate-member --account-id ${var.accounts.shared.id} --profile ${var.profile} --region ${var.region}"
+  }
+}
+
+resource "null_resource" "enable_member_account" {
+  provisioner "local-exec" {
+    command = "aws inspector2 enable --account-ids ${var.accounts.shared.id} --resource-types ECR EC2 --profile ${var.profile} --region ${var.region}"
+  }
+}
+
+###
+# Member account config, first disable next disassociate.
+###
+# resource "null_resource" "disable_member_account" {
+#   provisioner "local-exec" {
+#     command = "aws inspector2 disable --account-ids ${var.accounts.shared.id} --resource-types ECR EC2 --profile ${var.profile} --region ${var.region}"
+#   }
+# }
+
+# resource "null_resource" "disassociate_account" {
+#   provisioner "local-exec" {
+#     command = "aws inspector2 disassociate-member --account-id ${var.accounts.shared.id} --profile ${var.profile} --region ${var.region}"
+#   }
+# }
```

Comment:
@damianleys as per our discussion along @diego-ojeda-binbash and @fgauchat-binbash It could be great if we add a terraform bool flag type (true / false) in order to enabled and disable inspector in the different accounts. So these commented blocks could be more dynamic instead of commenting and uncommenting the code to do that.

CC: @binbashar/leverage-ref-architecture-aws-admin @binbashar/leverage-ref-architecture-aws-dev